src/test/java/com/session/service/ClientTest.java

```diff
@@ -92,9 +123,9 @@ public void getSessionByEmail_shouldReturnExpectedSession() throws Exception {
     public void getSessionByID_timeoutExpired_shouldReturnNotFound() throws Exception {
         SessionCreated sessionCreated = client.createNewSession(EMAIL);
 
-        System.out.println("waiting a bit...");
-        Thread.sleep(SESSION_TIMEOUT_MS);
-        System.out.println("wait ended...");
+        // System.out.println("waiting a bit...");
+        // Thread.sleep(SESSION_TIMEOUT_MS);
+        // System.out.println("wait ended...");
```

Comment:
If this is not needed `delete`

Reply:
Left it in, only until I get to it in my next PR.